practice-one/src/assets/styles/bases/global.css

```diff
@@ -8,5 +8,5 @@
 
 .container-fluid {
   width: 1440px;
-  margin: 0 auto;
+  /* margin: 0 auto; */
```

Comment:
Why do you comment this line?

Reply:
Because initially I have two choices, one is to use padding of class header-section or second is to use margin of class container. Finally, I chose to use the first option so I decided to comment that line but after checking again, I realized I should use the second option, I will update again